janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/rest/RestElasticSearchClient.java

```diff
@@ -200,14 +205,21 @@ public void createMapping(String indexName, String typeName, Map<String,Object>
     }
 
     @Override
-    public IndexMapping getMapping(String indexName, String typeName) throws IOException{
-        Response response = performRequest(REQUEST_TYPE_GET, REQUEST_SEPARATOR + indexName + REQUEST_SEPARATOR + "_mapping" + REQUEST_SEPARATOR + typeName, null);
-        try (final InputStream inputStream = response.getEntity().getContent()) {
+    public IndexMapping getMapping(String indexName, String typeName) throws IOException {
+        InputStream inputStream = null;
+        try {
+            final Response response = performRequest(REQUEST_TYPE_GET, REQUEST_SEPARATOR + indexName + REQUEST_SEPARATOR + "_mapping" + REQUEST_SEPARATOR + typeName, null);
+            inputStream = response.getEntity().getContent();
             final Map<String, IndexMappings> settings = mapper.readValue(inputStream, new TypeReference<Map<String, IndexMappings>>() {});
             if (settings == null) {
                 return null;
             }
             return settings.get(indexName).getMappings().get(typeName);
+        } catch (final JsonParseException | JsonMappingException | ResponseException e) {
+            log.debug("Error when we try to get ES mapping : " + e.getMessage(),e);
```

Comment:
Should this be logged at error or higher than debug? Also since you're logging the full exception I don't think you need the message.